src/librbd/crypto/BlockCrypto.cc

```diff
@@ -2,7 +2,9 @@
 // vim: ts=8 sw=2 smarttab
 
 #include "librbd/crypto/BlockCrypto.h"
+#if defined(__linux__)
```

Comment:
i'd suggest use `#include <stdlib.h>` instead.

Reply:
> 
> 
> i'd suggest use `#include <stdlib.h>` instead.

Right, if that is available on Linux in that include, I think we can remove the line.
Because on FreeBSD is looks like stdlib is also already previously included.
I'll try and we'll see if it compiles.

Reply:
> I think we can remove the line.

please don't.

see https://google.github.io/styleguide/cppguide.html#Names_and_Order_of_Includes

> You should include all the headers that define the symbols you rely upon, except in the unusual case of forward declaration. If you rely on symbols from bar.h, don't count on the fact that you included foo.h which (currently) includes bar.h: include bar.h yourself, unless foo.h explicitly demonstrates its intent to provide you the symbols of bar.h.

Reply:
@tchaikov 
Right, I was misled by the reorganizing effort of some time ago, where we really pruned all header including. And I expect that this clause could have been violated due to excessive cleaning.

fixed the PR.